dts/bindings/memory-controllers/st,stm32-fmc-nor-psram.yaml

```diff
@@ -0,0 +1,194 @@
+    st,timing:
+      type: array
+      required: true
+      description: |
+        SRAM/NOR-Flash (read) timing register (FMC_BTRx).
+
+        If the EXTMOD is set (see control register FMC_BCRx), then
+        FMC_BTRx register is partitioned for write and read access.
+        That means, use this property to configure read accesses and
+        'st,timing-ext' to configure write accesses.
+
+        Expected fields, in order:
+
+        * ADDSET  - Address setup phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the address setup time. This parameter can be a value
+                    between Min_Data = 0 and Max_Data = 15.
+                    Note: Not used with synchronous NOR Flash memories.
+        * ADDHLD  - Address-hold phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the address hold time. This parameter can be a value
+                    between Min_Data = 1 and Max_Data = 15.
+                    Note: Not used with synchronous NOR Flash memories.
+        * DATAST  - Data-phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the data setup time. This parameter can be a value
+                    between Min_Data = 1 and Max_Data = 255.
+                    Note: Used for SRAMs, ROMs and asynchronous multiplexed
+                    NOR Flash memories.
+        * BUSTURN - Bus turnaround phase duration.
+                    Number of HCLK cycles to configure the duration of
+                    the bus turnaround. This parameter can be a value
+                    between Min_Data = 0 and Max_Data = 15.
+                    Note: Only used for multiplexed NOR Flash memories.
+        * CLKDIV  - Clock divide ratio (for FMC_CLK signal).
+                    Period of CLK clock output signal, expressed in number of
+                    HCLK cycles. This parameter can be a value
+                    between Min_Data = 2 and Max_Data = 16.
+                    Note: Not used for asynchronous NOR Flash, SRAM or ROM
+                    accesses.
+        * DATLAT  - Data latency for synchronous memory.
+                    Number of memory clock cycles to issue to the memory
+                    before getting the first data.
+                    The value depends on the memory type as shown below:
+                    - It must be set to 0 in case of a CRAM
+                    - It is don't care in asynchronous NOR, SRAM or ROM accesses
+                    - It may assume a value between Min_Data = 2 and Max_Data = 17
+                      in NOR Flash memories with synchronous burst mode enable
+        * ACCMOD  - Access mode.
+                    See access mode defines
+                    in dt-bindings/memory-controller/stm32-fmc-nor-psram.h.
+
+    st,timing-ext:
+      type: array
+      required: false
+      default: [0xF, 0xF, 0xFF, 0xFF, 0x00]
```

Comment:
Use of `default` in the property must be justified in that property's description. Please see https://docs.zephyrproject.org/latest/guides/dts/bindings.html#default-values-for-properties and adjust accordingly.

Reply:
The driver makes use of `DT_PROP_BY_IDX` which uses `DT_PROP`.  In such a case, the build fails when there is no `default` value provided. That can be resolved with `DT_PROP_OR`. Unfortunately, that hides the default value in the driver's code.

Added more comments. That shall clarify the default value.